doc/releases/release-notes-4.0.rst

```diff
@@ -575,6 +742,10 @@ Drivers and Sensors
   * Added ESP32-C2 Wi-Fi support.
   * Added ESP32 driver APSTA support.
   * Updated ESP32 Wi-Fi driver to reflect actual negotiated PHY mode.
+  * Added WPS support for STA mode
+  * Added enterprise security support for STA mode
+  * Added regulatory domain configuration support
+  * Enabled time checking for mbedtls cert files
```

Comment:
you should really focus on changes that are related to NXP Platforms, not changes contributed by NXP employees, otherwise you're basically doing the work of other areas' maintainers and, more importantly, going to get yourself into lots of merge conflicts if other PRs (see e.g. https://github.com/zephyrproject-rtos/zephyr/pull/81216) get merged before yours..

Reply:
yeah, this point was brought up many times internally, I don't know why we are still doing this

Reply:
@yvanderv, I did a quick review of #81216 and it appears Jukkar generally captured the jist of these. We can remove them here.